Examples/Physics_applications/laser_ion/inputs

```diff
@@ -0,0 +1,194 @@
+#stop_time = 1.0e-12           # [s]
+
+# quick tests at ultra-low res. (CI)
+#amr.n_cell = 384 512
+
+# proper resolution for 10 n_c excl. acc. length
+# (>=1x V100)
+amr.n_cell = 2688 3712
+
+# proper resolution for 30 n_c (dx<=3.33nm) incl. acc. length
+# (>=6x V100)
+#amr.n_cell = 7488 14720
+
+# simulation box, no MR
+#   note: increase z (space & cells) for converging ion energy
+amr.max_level = 0
+geometry.prob_lo = -7.5e-6 -5.e-6
+geometry.prob_hi =  7.5e-6 25.e-6
+geometry.is_periodic = 0 0 0  # non-periodic (default)
+
+# macro-particle shape
+interpolation.nox = 3
+interpolation.noy = 3
+interpolation.noz = 3
+
+# numerical tuning
+warpx.cfl = 0.999
+warpx.use_filter = 1          # bilinear current/charge filter
+
+
+#################################
+# Performance Tuning
+#
+amr.max_grid_size = 64
+amr.blocking_factor = 64
+warpx.do_dynamic_scheduling = 0
+
+# load balancing
+warpx.load_balance_int = 10
+warpx.load_balance_with_sfc = 0
+#warpx.load_balance_efficiency_ratio_threshold = 1.1
+algo.load_balance_costs_update = Heuristic
+#algo.costs_heuristic_particles_wt = 0.9
+#algo.costs_heuristic_cells_wt = 0.1
+
+# particle bin-sorting on GPU (ideal defaults not investigated in 2D)
+#warpx.sort_int = 4
+#warpx.sort_bin_size = 4 4 4
+
+
+#################################
+# Target Profile
+#
+particles.species_names = electrons hydrogen
+
+# particle species
+hydrogen.species_type = hydrogen
+hydrogen.injection_style = NUniformPerCell
+hydrogen.num_particles_per_cell_each_dim = 2 2 4
+hydrogen.momentum_distribution_type = constant
+hydrogen.ux = 0.0
+hydrogen.uy = 0.0
+hydrogen.uz = 0.0
+#hydrogen.zmin = -10.0e-6
+#hydrogen.zmax =  10.0e-6
+hydrogen.profile = parse_density_function
+
+electrons.species_type = electron
+electrons.injection_style = NUniformPerCell
+electrons.num_particles_per_cell_each_dim = 2 2 4
+electrons.momentum_distribution_type = "gaussian"
+electrons.ux_th = .01
+electrons.uz_th = .01
+#electrons.zmin = -10.0e-6
+#electrons.zmax =  10.0e-6
+
+# ionization physics (field ionization/ADK)
+#   [i1] none (fully pre-ionized):
+electrons.profile = parse_density_function
+#   [i2] field ionization (ADK):
+#hydrogen.do_field_ionization = 1
+#hydrogen.physical_element = H
+#hydrogen.ionization_initial_level = 0
+#hydrogen.ionization_product_species = electrons
+#electrons.profile = constant
+#electrons.density = 0.0
+
+# collisional physics (binary MC model after Nanbu/Perez)
+#collisions.collision_names = c_eH c_ee c_HH
+#c_eH.species = electrons hydrogen
+#c_ee.species = electrons electrons
+#c_HH.species = hydrogen hydrogen
+#c_eH.CoulombLog = 15.9
+#c_ee.CoulombLog = 15.9
+#c_HH.CoulombLog = 15.9
+
+# number density: "fully ionized" electron density as reference
+#   [material 1] cryogenic H2
+my_constants.nc    = 1.742e27  # [m^-3]  1.11485e21 * 1.e6 / 0.8**2
+my_constants.n0    = 30.0      # [n_c]
+#   [material 2] liquid crystal
+#my_constants.n0    = 192
+#   [material 3] PMMA
+#my_constants.n0    = 400
```

Comment:
```suggestion
#my_constants.n0    = 230
```
I am not sure that the electron density of fully ionized [PMMA](https://en.wikipedia.org/wiki/Poly(methyl_methacrylate)) is 400 nc for a 800 nm laser. Considering a density of ~ 1.18 g/cm3 and a composition of (C5 O2 H8) I get ne ~ 230 nc ...

Reply:
Absolutely, I remembered this once for 1054nm and since then always mix it up. 230 is absolutely correct, thanks for noticing this.